presto-docs/src/main/sphinx/release/release-339.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+Release 339 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Add :func:`approx_most_frequent`. (:issue:`3425`)
+* Physical bytes scan limit for queries can be configured via ``query.max-scan-physical-bytes`` configuration property
```

Comment:
This should be documented in the query properties section and then linked to from here.

Reply:
Agreed, but that's out of scope for this PR. Once the documentation is added, we can link to it here.